tests/test_train_classifier.py

```diff
@@ -0,0 +1,280 @@
+# test_train_classifier.py
```

Comment:
Nice file!

Out of curiosity, why this comment `# test_train_classifier.py`?

Reply:
No good reason haha force of habit, for some pre-commit setups you need a comment at the beginning of the file, so I start by a dummy one. will take off